README.md

```diff
@@ -52,6 +52,26 @@ make VERSION=0.3.0 push
 make clean
 ```
 
+On Windows, using the tools described below, use the command:
+
+"C:\Program Files\git\bin\bash" -c "make"
+"C:\Program Files\git\bin\bash" -c "make test"
+"C:\Program Files\git\bin\bash" -c "make test TESTARGS='-run TestSomething'"
+"C:\Program Files\git\bin\bash" -c "make gofmt"
+
+
+## Installed requirements
+
+You'll need:
+* docker-ce (will work with move versions and platforms)
+* gnu make
+* golang
+
+On windows the building is somewhat more difficult due to the build being bash/linux/make-oriented, but support is provided. You need:
```

Comment:
- Choco itself needs to be installed if packages are being installed from it
- Installing choco, and from what I see, installing packages, requires running from an elevated command prompt.